packages/ActionBar/stories/ShowcaseApp/MySort.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Sort, useSort } from "../../src";
+
+export default function MySort({ columns }) {
+  const { appliedNumber, sortedFields, onAddField, onDeleteField, onChangeField, onApply } = useSort({
+    columns,
+  });
+
+  const handleDelete = fieldId => () => {
+    onDeleteField(fieldId);
+  };
+
+  const handleChange = fieldId => params => {
+    onChangeField({ ...params, id: fieldId });
+  };
+
+  return (
+    <Sort appliedNumber={appliedNumber} columns={columns} onAddField={onAddField} onApply={onApply}>
+      {sortedFields.map((field, index) => (
+        <Sort.Field
+          key={field.id}
+          {...field}
```

Comment:
maybe would be good to explain or add a comment what does field has inside so whoever is reading this code on the future can have an idea why you are spreading the `{...field}`